lib/paperclip/storage/azure.rb

```diff
@@ -166,30 +152,26 @@ def azure_interface
       end
 
       def obtain_azure_instance_for(options)
-        if options[:use_development_storage]
-          service = ::Azure::Storage::Blob::BlobService.create(use_development_storage: true)
-        else
-          service = ::Azure::Storage::Blob::BlobService.create(storage_account_name: options[:storage_account_name],
-                                                               storage_access_key: options[:storage_access_key])
-        end
-        
-        service
+        AzureBlob::Client.new(
+          account_name: options[:storage_account_name],
```

Comment:
Is it fine to pass these as `nil`? I assume so.

Reply:
`account_name` and `container` are required.
`access_key` is required if `principal_id` or `use_managed_identities` are both nil/false.
`cloud_regions` can be nil